.github/workflows/validate.yml

```diff
@@ -66,11 +66,7 @@ jobs:
       GHC_FOR_RELEASE: ${{ format('["{0}"]', env.GHC_FOR_RELEASE) }}
     strategy:
       matrix:
-<<<<<<< HEAD
-        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-=======
-        os: [ubuntu-latest, macos-13, windows-latest]
->>>>>>> d36e0d097 (CI: force MacOS jobs to use Intel runners (`macos-13`))
+        os: ["ubuntu-latest", "macos-13", "windows-latest"]
         # If you remove something from here.. then add it to the old-ghcs job.
         ghc: ["9.8.2", "9.6.4", "9.4.8", "9.2.8", "9.0.2", "8.10.7", "8.8.4", "8.6.5"]
```

Comment:
This will cause a conflict if we make changes later, because in `master` it uses single quotes (due to *this* PR, which is very confusing).

Reply:
Hm, actually no, it's due to the other backport. Which is conflicting with… itself, somehow.